CONTRIBUTING.md

```diff
@@ -11,8 +11,8 @@ Submit issues [here](https://github.com/Kotlin/kotlinx-kover/issues).
 * Search for existing issues to avoid reporting duplicates.
 * When submitting a bug report:
   * Test it against the most recently released version. It might have been already fixed.
-  * By default, we assume that your problem reproduces in Kotlin/JVM project with the Kover Coverage Tool. Please, mention if the problem is
```

Comment:
Initially, the wording was not quite correct. What was meant here: unless otherwise stated, it is assumed that the Kotlin/JVM plugin and Kover tool were used in the project.